client/actions/tests/test_show-heartbeat.js

```diff
@@ -195,13 +197,23 @@ describe('ShowHeartbeatAction', () => {
       revision_id: 42,
       arguments: {
         surveyId: 'my-survey',
+        includeTelemetryUUID: true,
```

Comment:
I think this new behavior should be put in a new test, instead of modifying an existing test.